src/string_view.h

```diff
@@ -22,11 +22,6 @@
 #include <lcf/dbstring.h>
 #include <fmt/core.h>
 
-// Needed to allow building with fmt 5, older versions are untested.
```

Comment:
This makes us remove retropie support, which seems still based on buster... #2719
Maybe let it stay until mid/end 2024?

Reply:
Does Buster support a recent gcc with c++17 support? If not this doesn't matter anyway.

Otherwise: Add a comment so it gets deleted right after 0.8.1 :)

Reply:
Version 8.3, which supports most features: https://en.cppreference.com/w/cpp/compiler_support/17